src/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,11 @@
+.layout {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+  
+  .content {
+    flex: 1;
+    padding: 20px;
+    margin-block: 40px;
+  }
```

Comment:
To have the content of the page fit neatly (no whitespace, all text fully visible), you should remove the padding and margin-block from `.content` and add the following rule:

```
.content > :first-child {
  padding-bottom: 20px;
  margin-block-start: 75px;
}
```